Source/Menu/MainForm.cs

```diff
@@ -1158,28 +1159,26 @@ void ShowDetails()
             if (radioButtonModeTimetable.Checked)
             {
                 if (SelectedTimetableSet != null)
-                {
                     ShowDetail(catalog.GetStringFmt("Timetable set: {0}", SelectedTimetableSet), new string[0]);
-                }
+                // Description not shown as no description is available for a timetable set.
+
                 if (SelectedTimetable != null)
-                {
-                    ShowDetail(catalog.GetStringFmt("Timetable: {0}", SelectedTimetable), new string[0]);
-                }
+                    ShowDetail(catalog.GetStringFmt($"Timetable: {SelectedTimetable}"), SelectedTimetable.Briefing.Split('\n'));
```

Comment:
```suggestion
                    ShowDetail(catalog.GetStringFmt("Timetable: {0}", SelectedTimetable), SelectedTimetable.Briefing.Split('\n'));
```
For Gettext catalog calls, it's best to stick with the existing syntax - changing it will break our translations.

Reply:
Good catch. I didn't realise GetStringFmt() had trouble with $""